content/de/cloud_computing.md

```diff
@@ -0,0 +1,24 @@
+---
+title: Vorlage für Definition
+status: Feedback erwünscht
+category: Konzept
+---
+
+## Was es ist
+Das Konzept, hinter dem Cloud Computing ist die Bereitstellung von Systemressourcen wie CPU, Netzwerk und Speicher auf Abruf und über das Internet.
+Das Cloud Computing ermöglicht den Zugriff und die Nutzung von Systemressourcen aus beliebigen Standorten.
+Diese Systemressourcen werden von Cloud-Anbietern wie AWS, GCP, Azure, Digitial Ocean, und anderen, aus unterschiedlichen Standorten, an den Endverbraucher vermietet.
```

Comment:
Ich bin mir nicht sicher ob vermietet hier das richtige Wort ist. Vielleicht sollten wir es in Anführungszeichen setzen?! @DaveVentura @Petria3s was denkt ihr?

Reply:
```suggestion
Diese Systemressourcen werden von Cloud-Anbietern wie AWS, GCP, Azure, Digitial Ocean und weiteren Anbietern an den Endverbraucher zur Verfügung gestellt.
```